Assets/Scripts/Player/MovementController.cs

```diff
@@ -188,7 +188,31 @@ void Update()
                     currentMovement.y * Vector3.Cross(camera.transform.right, Vector3.up).normalized +
                     currentMovement.x * Vector3.Cross(Vector3.up, camera.transform.forward).normalized;
                 // Move (without gravity). Whenever we move we also readjust the player's direction to the direction they are running in.
-                characterController.Move(absoluteMovement * Time.deltaTime);
+
+                //temporary to keep us from being stuck in the air
+                Tile _currentTile = null;
+
+                try
+                {
+                    _currentTile = GameObject.Find(string.Format("tile_{0}_{1}", Mathf.Round(transform.position.x), Mathf.Round(transform.position.z))).GetComponent<Tile>();
```

Comment:
Do not use GameObject.Find, especially not in `Update()`. It is a slow method to call, and hard coding the name is very inflexible way of coding.

Reply:
Also you can use string interpolation instead of `string.Format` for better readability